spec/models/repo_spec.rb

```diff
@@ -26,6 +26,45 @@
     end
   end
 
+  describe '#regex_branches' do
+    let(:branch_names) {
+      %w( 
+         develop 
+         master 
+         feature/#123-test 
+         feature/#456-test-2 
+         config_changes 
+         l10n_develop 
+      ) 
+    }
+    let(:regex) { raise NotImplemented }
+    let(:result) { subject.regex_branches(regex ) }
+ 
+    before do
+      allow(subject).to receive(:branch_names).and_return(branch_names)
+    end
+    context 'when it\'s a literal string' do
+      let(:regex) { /l10n_develop/ }
+      it 'returns just that branch' do
+        expect(result).to contain_exactly('l10n_develop')
```

Comment:
This won't match just `i10n_develop`, since it's not anchored (with `^` or `$`). Maybe add them if not present?